osinfo.go

```diff
@@ -14,26 +14,31 @@ type Release struct {
 	Arch    string
 	Name    string
 	Version string
-	Windows windowsRelease
-	Linux   linuxRelease
-	BSD     bsdRelease
+	Windows WindowsRelease
+	Linux   LinuxRelease
+	BSD     BsdRelease
+	MacOs   MacOsRelease
 }
 
-type windowsRelease struct {
+type WindowsRelease struct {
```

Comment:
This struct should not be exported. This is accessible with `release.Windows.<somekey>`